generator.yml

```diff
@@ -1,5 +1,5 @@
 llvm_versions:
-  - &llvm_tot 13
+  - &llvm_tot 14
   - &llvm_latest 12
```

Comment:
Do you want to add anchors for `llvm_12` and bump `llvm_latest` to 13?

Reply:
Not yet, TuxSuite does not have clang-13 yet: https://gitlab.com/Linaro/tuxsuite/-/issues/129

I can change the `llvm_latest` anchor to `llvm_12` and update all of the builds accordingly with this pull request then add `llvm_latest` back as `13` when the images are available? Choice is up to you.